### ISSUE TITLE
Concatenate two dataframes with same columns

### ISSUE THREAD

**Issue by Nicinic** [cardillo/joinery#83]
Hi,
I'm using Joinery on a project to compute statistics inside a simulator. I'm looking for the best way to concatenate two dataframes that have the same columns (since they were extracted from the same main dataframe). However, I haven't found a way to do this with joinery, and I kindadoubt that adding each row of the second dataframe to the first is a good solution to this problem. Is there a method to concatenate two dataframes, similar to the method "concat" in pandas ?
Thank you

**Comment by cardillo** [comment]
Does [append](http://cardillo.github.io/joinery/v1.9/api/reference/joinery/DataFrame.html#append(java.util.List%3C?%20extends%20V%3E)) give the desired result?

**Comment by Nicinic** [comment]
Yes it is possible to iterate over the second dataframe and append each row to the first dataframe to obtain the desired result
`
DataFrame<Object> df = ... ;
DataFrame<Object> df2 = ...;		
for(List<Object> row : df2) {
		df.append(row);
}
`

But is it really the optimal way to do it ?

**Comment by cardillo** [comment]
Agree, not ideal, but should get you going today.  I'll look at adding a more efficient `concat` implementation to next release.

**Comment by Nicinic** [comment]
Thanks. I will use this solution until the new version comes out.  And thanks a lot for this library

**Comment by jihoonkang0829** [comment]
Hi, I would like to contribute. Can you assign it to me? If not, I will start working on it!

**Comment by donggua223344** [comment]
Can you assign it to me? I would like to contribute
